.vib/clickhouse/goss/goss.yaml

```diff
@@ -0,0 +1,47 @@
+file:
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+    filetype: directory
+    mode: "3777"
+  /opt/bitnami/clickhouse/etc/conf.d/00_default_overrides.xml:
+    mode: "0644"
+    filetype: file
+    exists: true
+    contains:
+      - zookeeper
+      - remote_servers
+      - logger
+  /bitnami/clickhouse/data:
+    mode: "2750"
+    filetype: directory
+    exists: true
+http:
+  http://clickhouse:{{ .Vars.service.ports.http }}:
+    status: 200
+    body:
+      - Ok
```

Comment:
I would probably remove this as we are already interacting with the HTTP port in Cypress and verifying it works as expected.